tour_site_scraper.rb

```diff
@@ -0,0 +1,47 @@
+require 'rubygems'
+require 'selenium-webdriver'
+require 'pry'
+
+# This options are for headless execution of the browser so that it don't need to load browser 
+# options = Selenium::WebDriver::Firefox::Options.new(args: ['-headless'])
+# driver = Selenium::WebDriver.for(:firefox, options: options)
+
+driver = Selenium::WebDriver.for :firefox
+# Generate the search url physically using any date, time and put here, we will make it dynamic later based on requirement
+driver.navigate.to "https://www.tour.ne.jp/j_air/list/?adult=1&air_type=2&arr_out=ITM&change_date_in=0&change_date_out=0&date_out=20211231&dpt_out=TYO&time_from_out=0600&time_to_out=0700&time_type_out=0"
```

Comment:
Long line like this can be broken into multiple lines.

Reply:
@antorkhan san, Thank you for the review. I will do that in next step implementations.